### ISSUE TITLE
Tensorflow 2.0: where is tensorflow.contrib.layers.apply_regularization and tensorflow.contrib.layers.l2_regularizer?

### ISSUE THREAD

**Issue by anandece4u** [tensorflow/tensorflow#48067]
<from tensorflow.contrib.layers import apply_regularization, l2_regularizer>
I am trying to run the example of VAE which uses above code. Need help how to update this for latest tensorflow version.

below is snippet of code where it is getting used

<def build_graph(self):

        self.construct_weights()

        saver, logits = self.forward_pass()
        log_softmax_var = tf.nn.log_softmax(logits)

        # per-user average negative log-likelihood
        neg_ll = -tf.reduce_mean(tf.reduce_sum(
            log_softmax_var * self.input_ph, axis=1))
        **# apply regularization to weights
        reg = l2_regularizer(self.lam)
        reg_var = apply_regularization(reg, self.weights)**
        # tensorflow l2 regularization multiply 0.5 to the l2 norm
        # multiply 2 so that it is back in the same scale
        loss = neg_ll + 2 * reg_var
        
        train_op = tf.train.AdamOptimizer(self.lr).minimize(loss)

        # add summary statistics
        tf.summary.scalar('negative_multi_ll', neg_ll)
        tf.summary.scalar('loss', loss)
        merged = tf.summary.merge_all()
        return saver, logits, loss, train_op, merged>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Ubuntu 18.04.5 LTS (GNU/Linux 5.4.0-1040-gcp x86_64)
- TensorFlow installed from (source or binary): using pip
- TensorFlow version: 2.4.1
- Python version: 3.8.3
- Installed using virtualenv? pip? conda?: pip

**Comment by Saduf2019** [comment]
@anandece4u 
Kindly open a [stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow) issue for this as it is not a bug or feature request, Please post this kind of support questions at Stackoverflow. There is a big community to support and learn from your questions.
Thanks!
